### DISCUSSION TITLE
as french i want to donate

### DISCUSSION THREAD

**Discussion by katian** [smplayer-dev/smplayer#481] [Q&A] [answered]
Hello,

as french i want to donate but when i'm choose France on PayPal site i must to create a PayPal account and i dont want.

:disappointed:

**Comment by smplayer-dev** [comment, marked as answer]
I think this form allow to donate with a credit card:
https://www.paypal.com/donate/?hosted_button_id=C8H3RJPMFKHXG

It's also possible to donate cryptocurrencies:
https://www.smplayer.info/contribute.php

**Reply by katian** [reply]
Thanks a lot @smplayer-dev 
Ethereum (ETH): 0x312e280C0aFCE11f83d00b912FbC5942C719ECc9
it's done

**Comment by katian** [comment]
@smplayer-dev why not accept stable coins like USDT or USDC ?

**Reply by smplayer-dev** [reply]
Ok, I've just added some:
```
USDC (ETH network): 0x312e280C0aFCE11f83d00b912FbC5942C719ECc9
USDT (ETH network): 0x312e280C0aFCE11f83d00b912FbC5942C719ECc9
BUSD (Binance Smart Chain): 0x0794601e145A3EFed0F45E4aE50778c2B2dF8acE
USDC (Binance Smart Chain): 0x0794601e145A3EFed0F45E4aE50778c2B2dF8acE
USDT (Binance Smart Chain): 0x0794601e145A3EFed0F45E4aE50778c2B2dF8acE
```

**Comment by katian** [comment]
hello, for those who want transactions free of charge : https://nano.org/en

have a nice day !
